app/models/workspace.rb

```diff
@@ -0,0 +1,27 @@
+# == Schema Information
+#
+# Table name: workspaces
+#
+#  id         :bigint(8)        not null, primary key
+#  name       :string
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#  user_id    :bigint(8)
+#
+# Indexes
+#
+#  index_workspaces_on_user_id  (user_id)
+#
+# Foreign Keys
+#
+#  fk_rails_...  (user_id => users.id)
+#
+
+class Workspace < ApplicationRecord
+  has_many :projects, dependent: :delete_all
```

Comment:
destroy!!!!

if workspace is deleted what will be with it's projects? do we need to delete them too? what about tasks for project?